### DISCUSSION TITLE
Retrieve currently logged in user

### DISCUSSION THREAD

**Discussion by dashrandom** [Budibase/budibase#2457] [Help] [answered]
Hello, I would like to create a filter for a specific table on the frontend according to the currently logged in user.

Is there an expression I can use for the data provider filter to reference the currently logged in user (against _id for example)?

**Comment by kevmodrome** [comment, marked as answer]
No need to filter, you can select the `Current User.TableName` in the Data Provider settings.

**Reply by dashrandom** [reply]
thanks, found it!
